### ISSUE TITLE
Fix official root addon

### ISSUE THREAD

**Issue by AlvaroBrey** [z2-dev/android_device_zuk_z2_plus#5]
The official root addon doesn't work, the relevant option in Developer Settings is greyed out even after flashing the addon.

Other root addons (SuperSU, MagiskSU) seem to work without any problems.

**Comment by AlvaroBrey** [comment]
`persist.sys.root_access` does not exist in a clean install + su addon.
After manually setting it to `1` in `system.prop`, the following happens:

SELinux is denying access to `/system/xbin/su` by `init` for some reason, which prevents the su daemon from starting:
```
[    5.872321] type=1400 audit(1013033.355:4): avc: denied { getattr } for pid=1 comm="init" path="/system/xbin/su" dev="mmcblk0p16" ino=3013 scontext=u:r:init:s0 tcontext=u:object_r:su_exec:s0 tclass=file permissive=0
```

If allowed `r_file_perms` via `sepolicy`, instead the `execute` permission is blocked, and that can't be allowed in `sepolicy` due to https://github.com/LineageOS/android_vendor_cm/blob/cm-14.1/sepolicy/su.te#L72 .

**Comment by AlvaroBrey** [comment]
Fixed by 310109f8422ffd022922a14eece756dad576f284, I'm an idiot.
Thanks @SuperLamic
